src/DssSpell.t.base.sol

```diff
@@ -3584,6 +3584,24 @@ contract DssSpellTestBase is Config, DssTest {
         }
     }
 
+    function _fixChronicleStaleness(address oracle) internal {
+        bytes32 pokeDataSlot = bytes32(uint256(4));
+        bytes32 pokeData     = vm.load(oracle, pokeDataSlot);
+        uint128 pokePrice    = uint128(bytes16(pokeData << 128));
+
+        uint32 expiresAt = 365 days * 100;
+
+        vm.store(oracle, pokeDataSlot, bytes32(uint256(expiresAt) << 128 | uint256(pokePrice)));
+    }
+
+    function _fixChronicleStaleness() internal {
+        address chronicleBtc = 0x24C392CDbF32Cf911B258981a66d5541d85269ce;
+        address chronicleEth = 0x46ef0071b1E2fF6B42d36e5A177EA43Ae5917f4E;
```

Comment:
Are there any other oracles that you intend to use?

Reply:
These are what we have and will use for the foreseeable future with Aggor, but will update when needed